src/test/cbmc/FunctionCall_Ret-Param/main.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// cbmc-flags: --unwind 10
```

Comment:
No `--unwinding-assertions`?

Reply:
Not for any reasonably small unwind value (I gave up at 1000). We might be able to add the flag if we set an upper bound to `x`.

Reply:
Oh, I see. This would be a good candidate for the RMC flag that allows you to opt-out from basic safety checks like `--unwinding-assertions`